x/hub-genesis/keeper/transfer_enable_ics4_wrapper.go

```diff
@@ -36,7 +36,7 @@ func (w ICS4Wrapper) SendPacket(
 	data []byte,
 ) (sequence uint64, err error) {
 	state := w.k.GetState(ctx)
-	if !state.OutboundTransfersEnabled {
+	if !state.CanonicalHubTransferChannelHasBeenSet() {
```

Comment:
why we switched from `OutboundransfersEnalbed` ?

Reply:
now it's the same. both fields are set together